capture-sdk/sdk/src/main/java/net/gini/android/capture/GiniCapture.java

```diff
@@ -771,6 +781,8 @@ public static class Builder {
         private boolean mFlashButtonEnabled;
         private boolean mIsFlashOnByDefault = false;
 
+        private String mGiniBankVersion = "";
+
```

Comment:
[Request] This is not needed.

Reply:
Thanks for pointing this out! Deleted it.